java/pom.xml

```diff
@@ -688,7 +688,6 @@
           <reportSet><!-- by default, id = "default" -->
             <reports><!-- select non-aggregate reports -->
               <report>javadoc</report>
-              <report>test-javadoc</report>
```

Comment:
@lidavidm just for my understanding: why this fixes the issue?

Reply:
Javadoc errors if it doesn't find any public classes to document, JUnit 5 means that test classes no longer need to be public, so the easiest solution is to just not document test code

Reply:
Got it. Thanks.